roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_predicates.py

```diff
@@ -98,7 +98,7 @@ def run(self, terms, variables=None, regions_enabled=True, short_version=None,
                 {'name': 'MatchInterPodAffinity'}
             ])
 
-        if short_version in ['1.5', '1.6']:
+        if short_version in ['1.5', '1.6', 'latest']:
```

Comment:
hm.. probably not, I changed idea when I was going to commit it. I will just hardcode "latest" to the version to use.